doc/tutorial.rst

```diff
@@ -918,19 +901,9 @@ tarball:
   done
   done
 
-
-.. note::
-  If your build of Charliecloud doesn't support internal SquashFS mounting you
-  will want to use :code:`pdsh` to mount the image. You can't use :code:`srun`
-  to run the mount command because once it exits the mount process is killed
-  by Slurm and you will get a "Transport endpoint is not connected" error. For
-  more details see Charliecloud issue
-  `#230 <https://github.com/hpc/charliecloud/issues/230>`_.
```

Comment:
I think it would be good to document this somewhere, if you don't like it here, perhaps a FAQ entry?

Reply:
Yeah, I agree. FAQ would be good. I was just wanting to keep manual mount/unmount out of the tutorial.